$MirOS: ports/plan9/kencc/patches/patch-utils_cc_cc_h,v 1.1 2006/03/31 03:50:20 tg Exp $
--- utils/cc/cc.h.orig	Thu Mar 30 23:34:27 2006
+++ utils/cc/cc.h	Thu Mar 30 23:55:24 2006
@@ -535,13 +535,13 @@ void*	allocn(void*, long, long);
 void*	alloc(long);
 void	cinit(void);
 int	compile(char*, char**, int);
-void	errorexit(void);
+void	errorexit(void) __attribute__((__noreturn__));
 int	filbuf(void);
 int	getc(void);
 long	getr(void);
 int	getnsc(void);
 Sym*	lookup(void);
-void	main(int, char*[]);
+int	main(int, char*[]);
 void	newfile(char*, int);
 void	newio(void);
 void	pushio(void);
